lib/stripe/account.ex

```diff
@@ -42,19 +42,30 @@ defmodule Stripe.Account do
     support_phone: :string,
     support_url: :string,
     timezone: :string,
-    transfers_enabled: :boolean
+    transfers_enabled: :boolean,
+    legal_entity: %{module: Stripe.Account.LegalEntity}
   }
 
   @singular_endpoint "account"
   @plural_endpoint "accounts"
 
+  @valid_create_keys [:country, :email, :managed, :legal_entity]
```

Comment:
It's based on the fields allowed for `create` in Stripe's API docs: https://stripe.com/docs/api/curl#create_account

Similar thing for `@valid_update_keys`.